driver/src/main/scala/api/DBMetaCommands.scala

```diff
@@ -127,4 +128,11 @@ trait DBMetaCommands { self: DB =>
       self, command, ReadPreference.primary
     ).map(_ => {})
   }
+
+  /**
+   * Returns 1.0 when the server is responding to commands.
```

Comment:
I was returning 1.0 to be consistent with the mongo documentation. Do you want me to change it to be a Boolean?